vaadin-charts-flow-parent/vaadin-charts-flow-demo/src/main/java/com/vaadin/flow/component/charts/examples/dynamic/ServerSideEvents.java

```diff
@@ -29,7 +29,7 @@
 import com.vaadin.flow.component.charts.model.XAxis;
 import com.vaadin.flow.component.charts.model.YAxis;
 import com.vaadin.flow.component.checkbox.Checkbox;
-import com.vaadin.flow.component.html.Label;
+import com.vaadin.flow.component.html.*;
```

Comment:
by the java code convention in Vaadin, we use direct import only.

Reply:
Done